Presentation.Web/app/components/reports/reports-overview.ts

```diff
@@ -18,7 +18,7 @@
         public mainGridOptions: kendo.ui.GridOptions;
         private canCreate: boolean;
         private storageKey = "report-overview-options";
-        private gridState = this.gridStateService.getService(this.storageKey);
+        private gridState = this.gridStateService.getService(this.storageKey,String(this.user.id));
```

Comment:
hvorfor er du nødt til at lave String(this.user.id) her og de andre går bare igennem?.. Det er jo en int?

Reply:
Ændret value til number